core/language/en-GB/Filters.multids

```diff
@@ -11,4 +11,5 @@ SystemTiddlers: System tiddlers
 ShadowTiddlers: Shadow tiddlers
 OverriddenShadowTiddlers: Overridden shadow tiddlers
 SystemTags: System tags
+StoryList: Story minus Advanced search
```

Comment:
Thanks for turning it into a PR. I'd perhaps call it:

`tiddlers in the story, except AdvancedSearch`